src/twisted/internet/defer.py

```diff
@@ -53,6 +53,13 @@ class TimeoutError(Exception):
 
 
 
+class NoCurrentResult(BaseException):
+    """
+    This error is raised when attempting to call extractCurrentResulton a
+    L{Deferred} that has not yet fired all of its callbacks.
```

Comment:
Typically, we say the Deferred fires or is fired.  The callbacks are just called, I think.  Unless the lingo has moved on while I wasn't paying attention.